multiqc/utils/search_patterns.yaml

```diff
@@ -7,6 +7,9 @@ adapterRemoval:
 afterqc:
     fn: '*.json'
     contents: 'allow_mismatch_in_poly'
+atropos:
+    fn: '*.json'
+    contents: '"program": "Atropos"'
```

Comment:
Can we guarantee that this will always be at the top of the file? If so, would be nice to add `num_lines: 3` to prevent searching massive json files.

Reply:
done